### ISSUE TITLE
How do I set --taxon via the command line?

### ISSUE THREAD

**Issue by vragh** [harta55/EnTAP#32]
Hi EnTAP devs,

Thank you for putting together this great tool!!

When I try and set `--taxon` via the command line like so `EnTAP <blah> --taxon my_taxon`, the tool prints an error:

```
PARSE ERROR: Argument: --taxon
             Couldn't find match for argument
```

I cannot set the `taxon` argument in the `INI` file because I will be running many different transcriptomes at once, and I don't want to generate separate `INI` files for each of these (doesn't make sense since all the other commands will be identical).

Also the documentation is a bit unclear on this, but how exactly is the line indicating the selection of `InterProScan` databases supposed to be formatted? Like this?
```
protein=pfam,panther,cdd,prositeprofiles,prositepatterns,smart,coils,mobidblite
```
Because that throws up an error: `InterProScan selected, but invalid databases input!`

I presume I am doing something wrong here, or must have misunderstood the documentation. Any help you could extend would be appreciated.

**Comment by harta55** [comment]
Currently, the taxon command is only used in the ini file. It makes sense to also allow this in the command line (having the command line override ini file potentially), I'll think about including that next release. 

That is the correct format. Try removing the 'mobidlite' database. this may be a bug that it is rejecting that database.

**Comment by Aciole-David** [comment]
> Hi EnTAP devs,
> 
> Thank you for putting together this great tool!!
> 
> When I try and set `--taxon` via the command line like so `EnTAP <blah> --taxon my_taxon`, the tool prints an error:
> 
> ```
> PARSE ERROR: Argument: --taxon
>              Couldn't find match for argument
> ```
> 
> I cannot set the `taxon` argument in the `INI` file because I will be running many different transcriptomes at once, and I don't want to generate separate `INI` files for each of these (doesn't make sense since all the other commands will be identical).
> 
> Also the documentation is a bit unclear on this, but how exactly is the line indicating the selection of `InterProScan` databases supposed to be formatted? Like this?
> 
> ```
> protein=pfam,panther,cdd,prositeprofiles,prositepatterns,smart,coils,mobidblite
> ```
> 
> Because that throws up an error: `InterProScan selected, but invalid databases input!`
> 
> I presume I am doing something wrong here, or must have misunderstood the documentation. Any help you could extend would be appreciated.

I have also received the interproscan error and got rid of it by unselecting mobidblite.

**Comment by vragh** [comment]
@Aciole-David @harta55 thank you for the feedback. I'll give this a try then!!

**Comment by harta55** [comment]
InterProScan error corrected in latest version 0.10.8
